gigantum/gigantum-client
# Skip work dir migrations on first install

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
Small PR that fixes an issue where the work dir migration code errors if this is the first time you are running and the work dir has not been completely set up yet. Now, sensitive file and dataset file cache migrations are skipped if those directories haven't even been created yet.

**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
Should be able to start the client without a working directory and have everything start up and initialize properly.
